### PR TITLE
add trackTime functionality

### PR DESCRIPTION
**Pull request by andreineculau**
in order to obsolete https://github.com/tobiipro/aws-util-firecloud/blob/master/src/lambda.js#L9-L22

   * depends on #5 [ ]
   * depends on #4 [ ]

### CODE REVIEW

File: src/listeners/log-to-console.js [hunk 1]
```diff
@@ -108,9 +108,14 @@ export let logToConsole = function(cfg = {}) {
     let msgFormat = '';
     let msgArgs = [];
     if (entry.msg) {
+      let {msg} = entry;
```
Comment (IanSavchenko) [inline src/listeners/log-to-console.js]: why not destruct `_duration`?

File: src/minlog.js [hunk 2]
```diff
@@ -103,4 +103,16 @@ export default class MinLog {
       listener({entry, logger: this, rawEntry});
     });
   }
+
+  async trackTime(label, fn) {
+    let entry = {
+      _timeStart: new Date()
+    };
+    this.debug(label, entry);
```
Comment (IanSavchenko) [inline src/minlog.js]: Why specifically `debug` and not something else?

Comment (andreineculau) [reply src/minlog.js]: Want to send the logging fn as an arg?

Comment (IanSavchenko) [reply src/minlog.js]: > Want to send the logging fn as an arg?

Naah. I would still leave a default value and then I would need to override it on every usage - not nice.

I would choose out of two options: 
1. Rename function to `debugTrackTime` and set it in stone (so-so).
2. Add a property to `MinLog` class like `trackTimeLevelName` with some reasonable default like `debug`.

Comment (andreineculau) [reply src/minlog.js]: or, since it's a question of levels, I added a `time` level

**Review by IanSavchenko — APPROVED** [review]

**Pull request merged**